Student/usecase/student_usecase.go

```diff
@@ -35,7 +38,7 @@ func (s *studentUseCase) Create(c context.Context, st *domain.Student) error {
 		existingStudent, err := s.studentRepository.GetByID(ctx, st.ID)
 		if err == nil {
 			if !reflect.DeepEqual(*existingStudent, domain.Student{}) {
-				return errors.NewConflictError(fmt.Sprintf("Student with ID %s already exists", st.ID))
+				return errors.NewConflictError(fmt.Sprintf(errorMessage, st.ID))
```

Comment:
this should be a different error message indicating the student already exists instead of does not exist